react-setup/src/pages/Popup/sheetapi.js

```diff
@@ -0,0 +1,105 @@
+//returns lists of slots available
+async function get_slots() {
+    let response = await fetch("https://spreadsheets.google.com/feeds/cells/1CT8YGYORrT-DXoIC4rmmzfvCklcbpm3S-O0yS8OcIH0/od6/public/basic?alt=json")
+    if (response.ok) {
+      let resp = await response.json();
+      let data = resp.feed.entry;
+      var formatteddata = {}
+      for(var key in data){
+          var def = data[key].title.$t;
+          if(!formatteddata[def.slice(1)]){formatteddata[def.slice(1)] = {}; }
+          formatteddata[def.slice(1)][def[0]] = data[key].content.$t;
+      };
+      var result = {}
+      for(var key in formatteddata){
+        if(!formatteddata[key]['F'] || formatteddata[key]['F'] == 'slot') continue;
+        result[formatteddata[key]['F']] = '';
+      }
+      return Object.keys(result);
+    } else {
+      alert("HTTP-Error: " + response.status);
+    } 
+  }
+  get_slots();
```

Comment:
is this function call necessary here?

Reply:
no, its not corrected in the latest push